content/300-accelerate/200-getting-started.mdx

```diff
@@ -118,12 +118,6 @@ If your Prisma version is below `5.2.0`, generate Prisma Client with the `--acce
 npx prisma generate --accelerate
 ```
 
-If your Prisma version is below `5.0.0`, generate Prisma Client with the `--data-proxy` option:
```

Comment:
Does Accelerate support Prisma <5? If so, we need to keep that around.

Reply:
Ah, you're right! Accelerate can be used with earlier versions, good catch!!

Added it back: https://github.com/prisma/docs/pull/5575/files/24c43960317d569ce4223740d31984bea20f7b81#r1452408617

Reply:
Minimum version: 4.16.1
Min version with interactive transactions: 5.1.1 or higher
https://www.prisma.io/docs/accelerate/getting-started#prerequisites